cx-portal/src/features/appSubscription/appSubscriptionApiSlice.ts

```diff
@@ -80,11 +121,34 @@ export const apiSlice = createApi({
     >({
       query: (body) => {
         const statusId = `statusId=${body.statusId}`
+        const offerId = `offerId=${body.offerId}`
         const sortingType = `sorting=${body.sortingType}`
         return {
           url: `/api/Apps/provided/subscription-status?size=${PAGE_SIZE}&page=${
             body.page
-          }&${body.statusId && statusId}&${body.sortingType && sortingType}`,
+          }&${body.statusId && statusId}&${body.offerId && offerId}&${
+            body.sortingType && sortingType
+          }`,
+        }
+      },
+    }),
+    fetchAppFilters: builder.query<AppFiltersResponse[], void>({
+      query: () => `/api/apps/provided`,
+    }),
+    fetchSubscriptionDetail: builder.query<
+      SubscriptionDetailResponse,
+      SubscriptionDetailRequestBody
+    >({
+      query: (body) => {
+        return {
+          url: `/api/apps/${body.appId}/subscription/${body.subscriptionId}/provider`,
+        }
+      },
```

Comment:
no need for `return` statement when arrow function has no other logic than simply return one JSON object, wrap it in round brackets. Here:
`(body) => ({ url: ... })`